src/Tribe/Cost_Utils.php

```diff
@@ -104,7 +104,7 @@ public function get_separators() {
 	 * return int|float|string
 	 */
 	public function maybe_replace_cost_with_free( $cost ) {
-		if ( '0' === (string) $cost ) {
+		if ( '0' === (string) $cost || '0.00' === (string) $cost ) {
```

Comment:
Maybe we should use `number_format()`  to avoid having to check for a bunch of different values.
```
if ( '0.00' === number_format( $cost, 2 ) )
```